app/flatpak-builtins-remote-modify.c

```diff
@@ -167,6 +169,13 @@ get_config_from_opts (FlatpakDir *dir, const char *remote_name, gboolean *change
       *changed = TRUE;
     }
 
+  if (opt_api_server_url)
+    {
+      g_key_file_set_string (config, group, "xa.api-server-url", opt_api_server_url);
```

Comment:
I'm worried I'm late to this and missing some context, but the name api-server-url (here and elsewhere) seems overly generic. This is specifically the entitlements API, right? Could we call it entitlements-server-url?

Reply:
Context/Bikeshed here: https://docs.google.com/a/endlessm.com/document/d/1zE_QbB6mtdhjH5bsFdf9kPrYvjKAMBRl4uSRYDt-BqQ/edit?disco=AAAAC8Pzh-s